app/scripts/translations.js

```diff
@@ -1,5 +1,11 @@
 angular.module('gettext').run(['gettextCatalog', function (gettextCatalog) {
 /* jshint -W100 */
 	gettextCatalog.setStrings('de', {"My Profile":"DE - My Profile"});
+    gettextCatalog.setStrings('en', {
+        "URGENCY": "News Value",
+        "Urgency": "News Value",
+        "Urgency stats": "News Value stats",
+        "urgency": "News Value"
```

Comment:
I see, but value isn't - so we will have 3 this same translations for different keys. Which is bad.